lib/item.rb

```diff
@@ -1,3 +1,8 @@
 class Item
+    attr_reader :name, :price
 
+    def initialize(info)
+        @name = info[:name]
+        @price = info[:price]
```

Comment:
The interaction pattern asks to clean the string (remove the dollar sign) and convert to a float!

Reply:
Great commit history - nice stuff Paul!